official/resnet/keras/README.md

```diff
@@ -0,0 +1,57 @@
+This forlder contains the Keras implementation of the ResNet models. For more 
+information about the models, please refer to this [README file](../README.md).
+
+Similar to the [estimator implementation](/official/resnet), the Keras 
+implementation has code for both CIFAR-10 data and ImageNet data.  To use 
+either dataset, Make sure that you have the latest version of TensorFlow 
+installed and 
+[add the models folder to your Python path](/official/#running-the-models),
+otherwise you may encounter an error like `ImportError: No module named 
+official.resnet`.
+
+## CIFAR-10
+
+Download and extract the CIFAR-10 data. You can use the following script:
+```bash
+python cifar10_download_and_extract.py
+# Then to train the model, run the following:
+python cifar10_main.py
+
+```
+
+After you download the data, specify the location with the `--data_dir` flag, 
+like:
+
+```python
+python keras_cifar_main.py --data_dir=/path/to/cifar
+```
+
+## ImageNet
+
+Download the ImageNet dataset and convert it to TFRecord format. 
+The following [script](https://github.com/tensorflow/tpu/blob/master/tools/datasets/imagenet_to_gcs.py)
+and [README](https://github.com/tensorflow/tpu/tree/master/tools/datasets#imagenet_to_gcspy)
+provide a few options.
+
+Once your dataset is ready, you can begin training the model as follows:
+
+```bash
+python keras_imagenet_main.py --data_dir=/path/to/imagenet
+```
+
+There are more flag options you can specify. See 
+[`keras_common.py`](keras_common.py) for full list of options.
+
+## Compute Devices
```

Comment:
Suggested rewrite for this section:

## Using multiple GPUs
You can train these models on multiple GPUs using `tf.distribute.Strategy` API. You can read more about them in this [guide](https://www.tensorflow.org/guide/distribute_strategy).

In this example, we have made it easier to use is with just a command line flag `--num_gpus`. By default this flag is 1 if TensorFlow is compiled with CUDA, and 0 otherwise.

+ --num_gpus=0: Uses `tf.distribute.OneDeviceStrategy` with CPU as the device.
+ --num_gpus=1: Uses `tf.distribute.OneDeviceStrategy` with GPU as the device.
+ --num_gpus=2+: Uses `tf.distribute.MirroredStrategy` to run synchronous distributed training across the those GPUs.

If you wish to run without `tf.distribute.Strategy`, you can do so by setting `--distribution_strategy=off`

Reply:
Done.